### PR TITLE
Currency: Adding build script and build_info.json file for tslib.

### PR DESCRIPTION
**Pull request by AbhiIBM**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by AbhiIBM** [comment]
Verification log : [testlog.txt](https://github.com/ppc64le/build-scripts/files/12145087/testlog.txt)

File: t/tslib/build_info.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "github_url":"https://github.com/microsoft/tslib.git",
+    "version": "2.6.0",
+    "default_branch": "main",
+    "package_dir": "t/tslib",
+    "build_script": "tslib_ubi_8.7.sh",
+    "docker_build": false,
+    "validate_build_script": true,
+
+    "2.6.*,2.5.*" : {
+        "build_script": "argparse_ubi_8.7.sh"
```
Comment (nikhil-kalbande) [inline t/tslib/build_info.json:13]: Could be typo in copy-paste.. Correct the filename

File: t/tslib/tslib_ubi_8.7.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+git checkout $PACKAGE_VERSION
+
+if ! npm install && npm audit fix && npm audit fix --force; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+if ! node ./test/runTests.js && node ./test/validateModuleExportsMatchCommonJS/index.js ; then
```
Comment (nikhil-kalbande) [inline t/tslib/tslib_ubi_8.7.sh:47]: @AbhiIBM What was reference point for using these commands for install and test steps. Can you please point to there

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: t/tslib/tslib_ubi_8.7.sh (hunk 2 above)
Comment (AbhiIBM) [reply t/tslib/tslib_ubi_8.7.sh:47]: I found the steps to test the package on its CI.yml (https://github.com/microsoft/tslib/blob/2.6.0/.github/workflows/CI.yml#L21)

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**